### ISSUE TITLE
The D of D-test on waterloo is only 0.93, how to improve it?

### ISSUE THREAD

**Issue by Xiaocong6** [zwx8981/DBCNN-PyTorch#8]
I fine-tuned the DBCNN on LIVE dataset. ( python DBCNN.py and I didn't change the code.)
I used the models saved in 'db_models' folder, ( net_params_best.pkl ) to test on waterloo.
The results are as follows:
![图片](https://user-images.githubusercontent.com/44869509/60651357-9daaee80-9e78-11e9-8e0b-f0bf530ea38f.png)

There is a gap with your accuracy given in paper, What's the possible reason for this, what should I do to improve it.

**Comment by zwx8981** [comment]
Since this repo was not tested on waterloo dataset yet. try to save more weights along different epochs and try it again. make sure you finetune on the entire live dataset, not just 80%

**Comment by Xiaocong6** [comment]
Ok, i got it. Thank you very much!
